samples/bluetooth/peripheral_ancs_client/README.rst

```diff
@@ -0,0 +1,284 @@
+      Pairing completed: xx:xx:xx:xx:xx:xx (random), bonded: 1
+
+#. After bonding, verify in nRF Connect that the Client Characteristic Configuration descriptors (CCCD) for ANCS Notification Source (0x120D) and ANCS Data Source (0xC6E9) are set to ``01 00``.
+
+Send an iOS notification to the application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following table shows the format of a notification that you can send to the example application:
+
+   +------------------+---------------+--------------------------+
+   | Field            | Example value | Interpretation           |
+   +==================+===============+==========================+
+   | Event ID         | 0             | Notification added       |
+   +------------------+---------------+--------------------------+
+   | Flags            | 18            | Positive/negative action |
+   +------------------+---------------+--------------------------+
+   | Category         | 06            | Email                    |
+   +------------------+---------------+--------------------------+
+   | Category count   | 02            |                          |
+   +------------------+---------------+--------------------------+
+   | Notification UID | 01 02 03 04   | 67305985 (0x4030201)     |
+   +------------------+---------------+--------------------------+
+
+1. In nRF Connect, set the value of ANCS Notification Source (characteristic 0x120D) to ``00 18 06 02 01 02 03 04`` and click Update.
+#. Verify that the UART data is received as follows::
+
+      Notification
+      Event:       Added
+      Category ID: Email
+      Category Cnt:2
+      UID:         67305985
+      Flags:
+       Positive Action
+       Negative Action
+
+Perform a notification action
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The received notification has two flags: Positive Action and Negative Action. This means that you can perform these two actions on the notification.
+
+The following table shows the format of the message that the application must send back to perform a notification action:
+
+   +------------------+---------------+-----------------------------+
+   | Field            | Example value | Interpretation              |
+   +==================+===============+=============================+
+   | Command ID       | 2             | Perform notification action |
+   +------------------+---------------+-----------------------------+
+   | Notification UID | 01 02 03 04   | 67305985 (0x4030201)        |
+   +------------------+---------------+-----------------------------+
+   | Action           | 00            | Positive                    |
+   |                  |               |                             |
+   |                  | 01            | Negative                    |
+   +------------------+---------------+-----------------------------+
+
+1. Press Button 3 to perform a positive action and verify that the ANCS Control Point (0xD8F3) is updated to ``02 01 02 03 04 00``.
```

Comment:
```suggestion
1. Press **Button 3** to perform a positive action and verify that the ANCS Control Point (0xD8F3) is updated to ``02 01 02 03 04 00``.
```

Reply:
Done.

Besides, ANCS Control Point is modified to a GUI label 'Apple Control Point'. It aligns with nRF Connect for Desktop.